lib/elixir/src/elixir_erl.erl

```diff
@@ -532,6 +547,13 @@ deprecated_chunk(Deprecated) ->
   ChunkData = term_to_binary({elixir_deprecated_v1, Deprecated}, [compressed]),
   [{<<"ExDp">>, ChunkData}].
 
+doc_value(Doc) ->
+  case Doc of
+    nil -> none;
+    false -> hidden;
+    Doc -> #{<<"en">> => Doc}
```

Comment:
@josevalim AFAIK, there is no way to specify the language of the documentation in the code. I chose to default to `"en"` for now, because I think this is something that needs more thought and would raise more questions if we wanted to tackle it now.

Reply:
Perfect.